source/dub/commandline.d

```diff
@@ -1236,7 +1241,29 @@ class RemoveCommand : FetchRemoveCommand {
 		auto package_id = free_args[0];
 		auto location = dub.defaultPlacementLocation;
 
-		dub.remove(package_id, m_version, location, m_forceRemove);
+		size_t resolveVersion(in Package[] packages) {
+			writeln("Select version of '", package_id, "' to remove from location '", location, "':");
+			foreach(i, pack; packages)
+				writeln(i, ". ", pack.version_);
```

Comment:
Nitpick - I like the following format better:

```
1) 0.9.20
2) 0.9.21
3) all versions
```
I.e. `)` instead of `.` and 1-based index.

So how about this:

```D
foreach (idx, option; packages.map!(p => p.version_.toString).chain(only("all versions")))
    writefln("%s) %s, idx + 1, option);
```